src/network/networkd-network-gperf.gperf

```diff
@@ -100,6 +100,7 @@ DHCP.UseMTU,                            config_parse_bool,
 DHCP.UseHostname,                       config_parse_bool,                              0,                             offsetof(Network, dhcp_use_hostname)
 DHCP.UseDomains,                        config_parse_dhcp_use_domains,                  0,                             offsetof(Network, dhcp_use_domains)
 DHCP.UseRoutes,                         config_parse_bool,                              0,                             offsetof(Network, dhcp_use_routes)
+DHCP.Anonymize,                         config_parse_bool,                              0,                             offsetof(Network, dhcp_anonymity)
```

Comment:
this same commit should also carry the addition of the bool to the Network structure...